x/concentrated-liquidity/model/msgs.go

```diff
@@ -26,12 +26,14 @@ func NewMsgCreateConcentratedPool(
 	denom0 string,
 	denom1 string,
 	tickSpacing uint64,
+	precisionFactorAtPriceOne sdk.Int,
 ) MsgCreateConcentratedPool {
 	return MsgCreateConcentratedPool{
-		Sender:      sender.String(),
-		Denom0:      denom0,
-		Denom1:      denom1,
-		TickSpacing: tickSpacing,
+		Sender:                    sender.String(),
+		Denom0:                    denom0,
+		Denom1:                    denom1,
+		TickSpacing:               tickSpacing,
```

Comment:
What is the recason for having `TickSpacing` by the user? Isn't it determined by `PrecisionFactorAtPriceOne` dynamically?

Reply:
No, I think we need to find some different verbiage to explain this since its really easy to get confused. This tick spacing is referring to the ticks that are authorized to be initialized, i.e if this number is 5, then only ticks 0, 5, 10 etc can be initialized. The tick spacing you are referring to is the prices that each and every tick reference, which changes based on k value

Reply:
`TickSpacing` does not seem to be sued anywhere other than validation in `withdrawPosition` and `createPosition`

Reply:
I think it actually serves an important use case! Let me give an example:

Imagine we have a BTC<>USD pool trading in the $XXXX.XX range, and we want to trade at 10 cent ticks. Instead of launching the pool with with a kAtPriceOne value of -5, we can instead launch a BTC<>USD pool with tick spacing of 10 and a kAtPriceOne value of -6. This way, it gives us maneuver space if BTC were to go to $100,000. Instead of having to create a whole new pool, we can just change the tick spacing value to 1!

The problem with this is we cant easily bump this back up to 10 once we go to 1, so if we went from $100,000 to say $90,000 we would have to make a new pool.

The way I forsee it, if an asset were to have some crazy pump this is just a "fail-safe" for a lack of a better term that we can trigger while we decide if we want to make a new pool.

Please lmk your thoughts!

Reply:
Yeah feels like gov param would make the most sense

Reply:
Hmm normally protocol governance shouldn't be voting on individual pool parameters, but I can see the fallback scenario argument

As long as we limit to only being able to reduce tick spacing (to prevent a pool from being gov attacked) I don't see any security concerns, but it's still not 100% clear to me that this setup is a clean design

Reply:
Yeah I am not convinced either way yet, but its better to have the logic there and take it away instead of having to reimplement it later imo